runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -757,6 +757,11 @@ end)
 ---@param alias string language or filetype name
 ---@return string? # resolved parser name
 local function resolve_lang(alias)
+  -- validate that `alias` is a legal language
+  if not (alias and alias:match('[%w_]+') == alias) then
```

Comment:
Personally, I find dropping the parens makes the logic marginally better to follow.

```suggestion
  if not alias or alias:match('[%w_]+') ~= alias then
```

but feel free to ignore

Reply:
I wouldn't worry then.